app/views/groups/edit.html.erb

```diff
@@ -1,5 +1,5 @@
 <h1>
-  <%= @group.name %>
+  <%= render partial: "name", locals: {group: @group} %>
```

Comment:
Could this be made less verbose?
Maybe using view components: https://viewcomponent.org/guide/getting-started.html
```suggestion
  <%= render GroupNameComponent.new(group) %>
```

Reply:
Cool idea! But I think using this gem is a bit overkill in this case. Also, the rest of the project probably won't use it. Thus, if we wanted to include some item-layout, we would have two different ways of including partial layouts in our code.
Another less verbose way of including the partial would be
`<%= render "name", group: @group %>`

Reply:
I think the notifications team has already been using it

Reply:
Resolved by 6f2811f